docs/source/tutorials/chat.rst

```diff
@@ -132,9 +142,9 @@ as IDs 1 and 2. We can verify that these are our BOS and EOS tokens.
 
 .. code-block:: python
 
-    print(tokenizer.spm_model.piece_to_id("<s>"))
+    print(tokenizer._spm_model.spm_model.piece_to_id("<s>"))
```

Comment:
Should we add a quick comment to explain the nested spm_model attributes? As is it looks a little weird, but also don't wanna get into a whole digression about our tokenizer abstractions either

Reply:
not sure how to do this without requiring a long digression. maybe there's a better way to organize this in the tokenizer class